src/components/explorer/SearchBarFilters.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Card, HStack, Icon, useColorModeValue } from "@chakra-ui/react"
+import { motion } from "framer-motion"
+import { CircleWavyCheck, Sparkle, StarFour } from "phosphor-react"
+
+const ExplorerFilters = ["FEATURED", "NEWEST", "VERIFIED"] as const
+export type Filters = (typeof ExplorerFilters)[number]
+
+type Props = {
+  selected: Filters
+  onSelect: (filter: Filters) => void
+}
+
+const icons: { [K in Filters]: any } = {
+  FEATURED: StarFour,
+  VERIFIED: CircleWavyCheck,
+  NEWEST: Sparkle,
+}
+
+const SearchBarFilters = ({ selected, onSelect }: Props): JSX.Element => {
+  const selectedBg = useColorModeValue("white", "gray.600")
+  const selectedShadow = "0 0.5px 2px 0 rgba(0, 0, 0, 0.2)"
+
+  return (
+    <HStack as="ul" gap={1.5}>
+      {ExplorerFilters.map((filter) => (
+        <Button
+          key={filter}
+          leftIcon={<Icon as={icons[filter]} />}
+          as="label"
+          cursor="pointer"
+          borderRadius="lg"
+          alignSelf="center"
+          size="sm"
+          bgColor={"transparent"}
+          onClick={() => onSelect(filter)}
+          position={"relative"}
+          zIndex={1}
+          {...(selected === filter && { _hover: { bg: "transparent" } })}
+        >
+          {filter.toLowerCase()}
+          {selected === filter ? (
+            <motion.div
+              className="underline"
```

Comment:
irrelevant className left here from example. Also, should only have one MotionBox or MotionCard element instead of having motion.div with the Card child. We can make any component a motion one like `const MotionCard = motion(Card)`. That way we can use Chakra props here too